### ISSUE TITLE
AllianceWarData (24335) needs to be updated

### ISSUE THREAD

**Issue by d3death** [clugh/coc-messages#2]
This is last known WarDonations i had that worked before coc changed encryption.

```
{
  "id":24335,
  "name": "WarDonations",
  "fields": [
    {"type": "INT"},
    {"type": "INT"},
    {"name": "ownTeam", "type": "WarTeam"},
    {"name": "enemyTeam", "type": "?WarTeam"},
    {"type": "?LONG"},
    {"type": "WarEvent[]"}  
  ]
}
```

There are some other issues (i think) with 24335 and proxy which i posted [here](https://github.com/clugh/coc-messages-python/issues/1)

**Comment by d3death** [comment]
AllianceWarData is what i called WarDonations

**Comment by clugh** [comment]
This was fixed in 84dcfc643da29ec082dc48c4a9764ab39711bd91.
